test/export_image_and_mask_test.py

```diff
@@ -6,37 +6,28 @@
 CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
 
 
-def test_basic_ct_feature_extraction():
-    import shutil
-
-    write_dir = os.path.join(CURRENT_DIR, "data", "temp_data")
-    if os.path.exists(write_dir):
-        shutil.rmtree(write_dir)
-
+def test_basic_ct_feature_extraction(tmp_path):
     image = np.zeros((128, 128), dtype=float)
     image[32:64, 32:64] = 1.0
 
     mask = np.zeros((128, 128), dtype=bool)
     mask[48:64, 48:64] = True
 
-    data = extract_features_and_images(
+    extract_features_and_images(
         write_features=True,
-        export_features=True,
+        export_features=False,
         write_images=True,
         export_images=False,
-        write_dir=write_dir,
+        write_dir=tmp_path,
         image=image,
         mask=mask,
         base_feature_families="statistics"
     )
 
     # Check that files exist.
-    file_names = [file for file in os.listdir(write_dir) if os.path.isfile(os.path.join(write_dir, file))]
+    file_names = [file for file in os.listdir(tmp_path) if os.path.isfile(os.path.join(tmp_path, file))]
```

Comment:
> Merge with 2.2.1 prior to merging with the master branch. Add minor changes to use os.path.join for OS-independent paths after merging.

@alexzwanenburg, `tmp_path` is a `pathlib.Path` object. `Path` objects have a convenient slash operator `/` for joining paths, and this is OS independent.

It does look a bit unusual, but it performs as well as `os.path.join()`, works for both `Path` objects and strings (so long as at least one part is a `Path` object) and is markedly easier to read, IMHO. There was no need to revert to `os.path.join()`.

If you really prefer a "joinpath" function, it would be cleaner to use `pathlib`'s `.joinpath()` method